Service/src/main/java/Service/Service/Item.java

```diff
@@ -0,0 +1,269 @@
+package Service.Service;
```

Comment:
nitpick: should we have a better package structure. Maybe models/ and controller/ directories?